### ISSUE TITLE
What is Progressbar and what is our vision for the future

### ISSUE THREAD

**Issue by olso** [Progressbar/Progressbar-Improvement-Proposal#56]
This is the current one

"Základným cieľom združenia je podpora rozvoja vedomostí a zručností v oblasti technológií, vedy, umenia a kultúry."

Does not really reflect anything, we need something that we can advertise

**Comment by yangwao** [comment]
We should focus on something what brings value and keep us at market.
Something, that people can actually pay for.

**Comment by yangwao** [comment]
https://www.facebook.com/agnes.pockels/posts/10213471635558780
text from web (if someone decides delete post)

> I’ve spent a big part of the last two years on rethinking how culture, art and community spaces can function outside of the default structures. The point was not to oppose the existing systems but to create a third way so we can realize again and again that there is always more possibilities of making things done, that the seemingly powerless crowd can be as impactful as a powerful individual. After 1 year of existence of blockchain-run project space called Temporary https://temporary.fi/ which was generously supported by Kone Foundation, we’ve moved to even more decentralized financial structure, with hopefully even more people involved. We’ve called it Kuusi Palaa. This name usually makes me awkward to pronounce it and it leaves any person, who knows at least a bit of Finnish, confused. Nevertheless, it describes the core of the whole idea very well: it’ is challenging, it can have many meanings depending on what do you actually want it to be.
> 
> Kuusi Palaa is a cooperative and a physical space which is co-owned and co-created by every member. Located in Helsinki’s Kallio, it offers public events programme collectively curated and produced, a digital currency/points system to run the space, 24/7 access and much more. Anybody who wants to be part of it please have a look on https://kuusipalaa.fi/, learn more about the whole idea, and consider to be part of the cooperative through obtaining your stake.
> 
> Without selling 100 stakes we will not be able to launch this idea on 1st of March as planned. So please, if you want to become a member don't leave for later your decision and spread the word. If you don’t want to be a member, no pressure, just keep making things as usual.

tl;dr; they made it, they failed, at least it boosted space somewhere.

**Comment by yangwao** [comment]
It would be nice to focus on technical people and crypto people involved in project.

We done Nodeschool in past. There is noodles.digital
We are developers.
We are into crypto as well.
We can try out tabletoken as our product. Cobot is co-up product, we can do the same. 💪

**Comment by anettrolikova** [comment]
I've been thinking bout this lately... the thing that we want to have space in rainbow color palette which is cool (example got me @Ivkaa https://githubuniverse.com/& https://ethereum.org/ ) .. have wall art in space is cool as well but does anyone was thinking about which company we want to have in space (that will have fix desks like decent back then or minitech nowdays) 🤔 🤔 🤔 cuz I'm not sure if in Bratislava there's company that we can focuse to get in our space... I know that having space full of led strips... wall art & blue & purple & pink colors sounds fun & will look cool but we need company that will be in our space & pay money but I just don't see who we want to target with this visual concept ... if someone doesn't understand me please feel free to ask me & don't judge me I'm just trying to look at it from other side.

**Comment by Ivkaa** [comment]
Hey, thank you for you food for thought. What I think is:
We want to be incubator as well, which would mean we want to support companies that would like space like that, even though maybe now in Bratislava there are very few. Also we need to be different than other coworks/etc. and this could be one of the things. I think many tech oriented people really like this kind of environment and it promotes creativity, not like corporate offices.
Btw your personal opinion about it is that you like it or not? :)
